src/main/java/frc/robot/RobotContainer.java

```diff
@@ -120,6 +120,8 @@ private void configureButtonBindings() {
     m_operator.getButtonB().whileHeld(new RunCommand(() -> m_loader.ManualLoad(0.5, 0.5), m_loader));
     m_operator.getButtonX().whileHeld(new RunCommand(() -> m_intake.ManualIntake(0.5), m_intake));
     m_operator.getButtonY().whileHeld(new RunCommand(() -> m_intake.ManualIntake(-0.5), m_intake));
+    m_operator.getButtonSelect().whileHeld(new RunCommand(() -> m_intake.MoveIntakeDown(), m_intake));
+    m_operator.getButtonStart().whileHeld(new RunCommand(() -> m_intake.MoveIntakeUp(), m_intake));
```

Comment:
For testing, we should comment this out until we're ready to actually test the pneumatic acuators.
Can we comment this out for now?

Reply:
We did this